### ISSUE TITLE
Add externals as a config parameter

### ISSUE THREAD

**Issue by harryluo91** [AnomalyInnovations/serverless-bundle#50]
Hi, would it be possible to add support for custom `externals` config option? I am currently using this with `sequelize` and have to add a conditionally required library to the `externals` in order for webpack to build. I am using my own fork now but it would be great to add this support. Thanks!

**Comment by jayair** [comment]
Yeah we could. But for certain obvious packages that can only be used through the `externals` option, we want to do it by default.

Can you share a link to the `sequelize` docs on how to use (or exclude) with Webpack?

**Comment by Phrosh** [comment]
You could keep the three externals that are already there and add and optional config for additional ones.
I just needed "chrome-aws-lambda" as external. You could add this to the repo, or simply enable users to add externals when needed.
